src/react-components/media-browser.js

```diff
@@ -282,6 +282,13 @@ class MediaBrowser extends Component {
     const activeFilter =
       searchParams.get("filter") || (searchParams.get("similar_to") && "similar") || (!searchParams.get("q") && "");
 
+    const meta = this.state.result && this.state.result.meta;
```

Comment:
might want to be careful here, if `result` is set but `meta` is not, then `meta === this.state.result`, so any keys on `result` that align with `meta` (now or ever) would cause bugs. i'd suggest ensuring `meta` is set to null in the case this expression is falsey.